build-tools/automation/yaml-templates/run-msbuild-device-tests.yaml

```diff
@@ -11,7 +11,7 @@ parameters:
 
 jobs:
   - job: ${{ parameters.job_name }}
-    displayName: MSBuild With Emulator - macOS-${{ parameters.node_id }} - ${{ parameters.job_suffix }}
+    displayName: macOS > Tests > MSBuild+Emulator ${{ parameters.job_suffix }} \#${{ parameters.node_id }}
```

Comment:
:rofl: , the `\` appears in the "alt/hover text" in AzDO.

I'll need to try *quoting* the line to allow `#` in there.